workflow-templates/yarn2-test.yml

```diff
@@ -80,8 +80,7 @@ jobs:
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: ${{ runner.os }}-yarn-
 
-      - name: Install dependencies (if not previously cached)
-        if: steps.yarn-cache.outputs.cache-hit != 'true'
```

Comment:
Fails sometimes because "package.json is not installed" while cache is recovered. Always `yarn install`. when cache is restored, it does just check if all packages are present. No need for this condition, imho.

Reply:
Ok from my side. Will only "slow down" execution by maybe 5sec (that is the only disadvantage I can think of)

Reply:
We need to check that everywhere then (also in yarn instllations and besides test also deploy and the other actions, ...)